mmv1/templates/terraform/examples/container_attached_cluster_full.tf.erb

```diff
@@ -0,0 +1,34 @@
+data "google_project" "project" {
+}
+
+resource "google_container_attached_cluster" "primary" {
+  name     = "<%= ctx[:vars]['name'] %>"
+  description = "Test cluster"
+  distribution = "aks"
+  annotations = {
+    label-one = "value-one"
+  }
+  authorization {
+    admin_users = [ "user1@example.com", "user2@example.com"]
+  }
+  oidc_config {
+      issuer_url = "https://oidc.issuer.url"
+      jwks = base64encode("{\"keys\":[{\"use\":\"sig\",\"kty\":\"RSA\",\"kid\":\"testid\",\"alg\":\"RS256\",\"n\":\"somedata\",\"e\":\"AQAB\"}]}")
+  }
+  platform_version = "1.24.0-gke.1"
+  fleet {
+      project = data.google_project.project.number
+  }
+  logging_config {
+	  component_config {
+      enable_components = ["SYSTEM_COMPONENTS", "WORKLOADS"]
+    }
+  }
+  monitoring_config {
+    managed_prometheus_config {
+      enabled = true
+    }
+  }
+  project = data.google_project.project.project_id
```

Comment:
by convention, the project & location usually go at the top of the resource, since they're foundational to identifying the resource. not a blocker.